kubernetes/reference/DC-kubernetes_rc_k3s-slemicro_lnvgy-se350-se450

```diff
@@ -0,0 +1,236 @@
+# uncomment DRAFT until content is completed/validated/ready for submission
+#DRAFT=yes
+ROLE="sbp"
+PROFROLE="sbp"
+
+## -------------------------------
+## Doc Config File for DAPS
+## Technical Reference Documentation
+## -------------------------------
+
+##
+# Basics
+##
+
+# Main Document 
+MAIN="SA.adoc"
+# Stylesheet Root
+STYLEROOT="/usr/share/xml/docbook/stylesheet/suse2021-ns"
```

Comment:
If you want to use the new stylesheets (not sure if this is applicable for this guide), use this:

```suggestion
STYLEROOT="/usr/share/xml/docbook/stylesheet/trd"
```